secboot/secboot.go

```diff
@@ -193,3 +195,26 @@ type UnlockResult struct {
 func EncryptedPartitionName(name string) string {
 	return name + "-enc"
 }
+
+// MarkSuccessful marks the secure boot parts of the boot as
+// successful.
+//
+//This means that the dictionary attack (DA) lockout counter is reset.
+func MarkSuccessful() error {
```

Comment:
Won't this function return an error on devices that are encrypted but without a TPM?